### DISCUSSION TITLE
Support DOM3 key codes (keyboard-layout-independent key inputs, e.g. W,A,S,D)

### DISCUSSION THREAD

**Discussion by parasyte** [melonjs/melonJS#1027] [Ideas]
This is supported since Firefox 32, and appears to be on-track for Chrome 41 according to: https://www.chromestatus.com/features/5228092293382144 and https://code.google.com/p/chromium/issues/detail?id=227231

Here's the description for the  `code` property: https://dvcs.w3.org/hg/dom3events/raw-file/tip/html/DOM3-Events.html#keys-codevalues

And here's the list of values: http://www.w3.org/TR/DOM-Level-3-Events-code/#code-value-tables

**Comment by obiot** [comment]
up for this one, as KeyboardEvent.code has been deprecrated for some time now, and it's definitely time to add support for [keyCode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)
